tests/azure/main.tf

```diff
@@ -0,0 +1,203 @@
+provider "azurerm" {
+  features {}
+  subscription_id = "XXXXXXXXXXXXXXXXXXXXXXXXXXX"
```

Comment:
Could you allow the option for these to be pased in via environment variables? https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#argument-reference. Just want to discourage hard coding secrets in files.

Reply:
Sure, will do it.

Reply:
I have updated the README file.